lambdas/file_access_patterns/lambda_for_fileUpload/index.js

```diff
@@ -8,17 +8,20 @@ function response(statusCode, message) {
     body: JSON.stringify(message),
   };
 }
+function generateTimestamp() {
+  const ts = Date.now();
+  const pattern = DT.compile("YYYY-MM-DD-HH-mm-ss");
```

Comment:
DT is not defined.
`const DT=require("date-and-time")` also do the npm install to include packages in package.json

Reply:
Noted